api/tests/core/educational/test_models.py

```diff
@@ -600,3 +602,11 @@ def test_unique_program_for_an_educational_institution(self):
         with pytest.raises(sa_exc.IntegrityError):
             institution.programs = [program1, program2]
             db.session.commit()
+
+
+class CollectiveOfferDisplayedStatusTest:
+    @pytest.mark.parametrize("status", set(CollectiveOfferDisplayedStatus))
+    def test_filter_statuses_but_one(self, status):
```

Comment:
Ce n'est pas vraiment ce qu'on test non ? 
```suggestion
    def test_get_offer_status(self, status):
```